src/App.js

```diff
@@ -6,13 +6,15 @@ import Register from './pages/register/register';
 import Group from './pages/group/group';
 import Swipe from './pages/swipe/swipe';
 import Upload from './pages/upload/upload';
+import Header from './components/header/header';
 
```

Comment:
add ./src/App.js to .gitignore so you dont push in the changes to the App.js files:
Do the following:

remove all the changes you made from App.js
push the changes from App.js
add App.js to gitignore -> now you can edit app.js however you want to
make any changes to your files (If you need to)
push your changes and pull request!